server/main.py

```diff
@@ -73,8 +87,9 @@ async def del_job(request: web.Request):
 
 app = web.Application()
 app.router.add_post('/add_service', add_service)
+app.router.add_post('/receive_alert', receive_alert)
```

Comment:
To be able to answer by clicking the link from the browser get endpoint is also needed.
```suggestion
app.router.add_post('/receive_alert', receive_alert)
app.router.add_get('/receive_alert', receive_alert)
```